astropy/timeseries/tests/test_downsample.py

```diff
@@ -33,38 +38,115 @@ def test_timeseries_invalid():
         aggregate_downsample(None)
     assert exc.value.args[0] == ("time_series should be a TimeSeries")
 
-    with pytest.raises(TypeError) as exc:
-        aggregate_downsample(TimeSeries())
-    assert exc.value.args[0] == ("time_bin_size should be a astropy.unit quantity")
 
+def test_time_bin_invalid():
 
-def test_downsample():
-    ts = TimeSeries(time=INPUT_TIME, data=[[1, 2, 3, 4]], names=['a'])
-    ts_units = TimeSeries(time=INPUT_TIME, data=[[1, 2, 3, 4] * u.count], names=['a'])
+    # Make sure to raise the right exception when time_bin_* is passed incorrectly.
```

Comment:
```suggestion
    ts = TimeSeries(time=INPUT_TIME, data=[[1, 2, 3, 4]], names=['a'])
    ts_units = TimeSeries(time=INPUT_TIME, data=[[1, 2, 3, 4] * u.count], names=['a'])
    # Make sure to raise the right exception when time_bin_* is passed incorrectly.
```
... and put them back in here.